webgl-obj-loader.js

```diff
@@ -113,12 +113,16 @@
 
       if (VERTEX_RE.test(line)) {
         // if this is a vertex
+        // DIscard optional [w] coordinate if present
+        elements.length = 3;
```

Comment:
What is the purpose of this? Setting the `length` attribute on elements here is never read. When dropping into the for loop when creating faces, `elements` will already be a different value.